types/node/v7/base.d.ts

```diff
@@ -2965,6 +2965,7 @@ declare module "fs" {
         mode?: number;
         autoClose?: boolean;
         start?: number;
+        highWaterMark?: number;
```

Comment:
You can skip updating 6 and 7 and 9 since they are not LTS.

Reply:
Should I revert the changes to 6, 7 and 9, or let them be?